<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tideCompare</class>
 <widget class="QMainWindow" name="tideCompare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tideCompare</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="time">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>3000</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>1900</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QPushButton" name="pushButtonPlot">
      <property name="text">
       <string>绘图</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_calVo">
      <property name="text">
       <string>计算</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpenFiles"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>状态</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFiles">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
